dotcom-rendering/docs/testing.md

```diff
@@ -74,6 +74,9 @@ The down side to these types of tests is that they are slower and have a depende
 By using mocked data and endpoints, we increase the speed that tests execute at and have complete certainty in what to expect from our mocked endpoints. The risk though is that an api might have changed or break and we won't be aware of this.
 
 ### How to run locally
+
+Running Cypress locally requires having a DCR server running. To run the CI server locally, you can run `make run-ci`. This command runs `make stop`, followed by `make build-ci` and `make start-ci`. You can run these separate commands yourself instead if you want more control over when the site gets re-built.
```

Comment:
> This command runs `make stop`, followed by `make build-ci` and `make start-ci`. You can run these separate commands yourself instead if you want more control over when the site gets re-built.

I wonder if adding the extra technical info without a use-case adds a little confusion to the really clear "To do this, do this" sentence before?

Not as part of this PR - If this is a requirement to run the below steps and we already have a `make cypress` - maybe we could have a make `dev-cypress-{open | run | mocked | e2e}`? Saying this I am unsure of where we want to use `yarn` vs `make`, but my understanding is that `make` was introduced to stop the need to remember running multiple things.

Reply:
that's a good point! I've removed the unnecessary details.

on the second point -- can I just check that we don't already have anything like the single command that you described @jamesgorrie ? I couldn't find one when I was looking, but perhaps I missed something?